config/config.json

```diff
@@ -2,28 +2,28 @@
   "development": {
     "username": "fiddle",
     "password": "fiddle",
-    "database": "fiddle_development",
+    "database": "fiddle",
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
   "test": {
     "username": "fiddle",
     "password": "fiddle",
-    "database": "fiddle_test",
```

Comment:
at least test env should have different DB name, or it's going to be very inconvenient to develop locally